end-to-end-tests/wdio/webdriver-manager.conf.js

```diff
@@ -35,18 +35,14 @@ exports.config = {
     // and 30 processes will get spawned. The property handles how many capabilities
     // from the same test should run tests.
     //
-    maxInstances: 1,
+    maxInstances: 5,
```

Comment:
:+1:

Reply:
@alisman Wondering how this relates to number of cores on physical machine.